lib/henson/source/path.rb

```
@@ -14,6 +24,25 @@ def valid?
       def path_exists?
         path && File.directory?(path)
       end
+
+      def version_from_modulefile
```

I'm kind of contemplating adding multiple DSL classes, one of which would be a `DSL::Modulefile`. In this case, we could just define `version` in that DSL and then set `method_missing` there to just silently ignore any other bits.